tools/get-deps/main.go

```diff
@@ -130,6 +130,9 @@ func parseDockerfile(f io.Reader) []string {
 		if err != nil {
 			panic(err)
 		}
+		if st.Platform != "" && st.Platform != targetArch {
```

Comment:
@rene suggested to push an empty riscv64 image to the docker hub and keep the depencies architecture-independent

PR for this is https://github.com/lf-edge/eve/pull/4061

once https://github.com/lf-edge/eve/pull/4061 is merged, this will be removed and this PR can be merged